### DISCUSSION TITLE
PKHEX Dev Build GIT

### DISCUSSION THREAD

**Discussion by mgibson041387** [kwsch/PKHeX#3925] [General]
Is there a GIT for the PKHEX Dev Build?

**Comment by sora10pls** [comment]
What? The “dev build” of PKHeX is just automated builds from the very GitHub repository you opened this discussion on.

If you want to download the compiled build itself (which is not "a GIT"), they can be found on the forums here: https://projectpokemon.org/home/files/file/2445-pkhex-development-build/
